### ISSUE TITLE
 Publish 'set' 'state' to '0xbc33acfffe5ec849' failed: 'Error: Command 0xbc33acfffe5ec849/1 genOnOff.on({},

### ISSUE THREAD

**Issue by domnich** [Koenkk/zigbee2mqtt#4709]
## What happened
Hello, I have a really small network of zigbee devices (8). 

I have problem with my wall socket:  zigbeeModel: ['TS0012'].

I started to observe strange error after added Router to my network (https://www.zigbee2mqtt.io/devices/TS0121_plug.html) 

The error from my zigbee2mqtt log is located here https://hastebin.com/lulaxomuru.sql.

I found a similar issue https://github.com/Koenkk/zigbee2mqtt/issues/4294. So first what I did was applying  same fix you provided, I  added disableDefaultResponse: true param  to my existing device,  but with out luck...

It also could be that my issue is similar to this one https://github.com/Koenkk/zigbee2mqtt/issues/4392 as here also discussed a Blitzwolf shp-13 power plug router.

And I also know that if I will totally remove my wall socket from mqtt and from zigbee db files and after will pair my wall switch again --> it will work as expected with out this error.
 

How I see it: If socket is broken and to solve the problem I just need to pair it again, could it be that zigbee2mqtt save and store some device related information somewhere in the storage ? 

Maybe the easiest fix for this problem will be just change Blitzwolf shp-13 power plug router to some other, BUT there is no certainty  that this will not happen with another router ..... especially when I have so few devices

Thanks, any information will be useful for me!

## Debug info
Zigbee2MQTT version: 1.15.0
Adapter hardware: сс2538 + сс 2592
Adapter firmware version:

**Comment by Koenkk** [comment]
Can you post your networkmap?

**Comment by domnich** [comment]
<img width="1000" alt="Screenshot at Oct 19 22-29-55" src="https://user-images.githubusercontent.com/5764719/96502576-aa647900-125a-11eb-86f1-5c101bacae7c.png">

**Comment by domnich** [comment]
I power off my Blitzwolf shp-13 for now as I was able to reproduce this issue 3 times to day...

One remark from my side. At the ticket description I wrote next: **And I also know that if I will totally remove my wall socket from mqtt and from zigbee db files and after will pair my wall switch again --> it will work as expected with out this error.**

I was wrong here, to fix this problem I just need to power off and then power on my Blitzwolf shp-13  router

**Comment by Koenkk** [comment]
If you could sniff the traffic we can dive deeper into this (https://www.zigbee2mqtt.io/how_tos/how_to_sniff_zigbee_traffic.html)

**Comment by domnich** [comment]
Thanks Koenkk. After my previous comment I added property "disableDefaultResponse: ture" into Blitzwolf shp-13 object to the "meta" (in devices.js) and for now everything works Ok...I can even say that my wall sockets now connected to this router. At the moment I can't say for sure that my fix  fixed the problem, as little time has passed. Now I want to watch and if the error will appears again I will necessarily sniff my traffic and provide it to you.

Koenkk could you please share some good resources with detailed info about zigbee device object model (e.g. disableDefaultResponse etc) ? I will be very grateful.

**Comment by domnich** [comment]
<img width="804" alt="Screenshot at Oct 21 10-33-03" src="https://user-images.githubusercontent.com/5764719/96687639-d8e07200-1388-11eb-8268-5677eece47ee.png">

**Comment by Shaquu** [comment]
@Koenkk I am not sure if I should open a new Issue. I have simillar error:
herdsman logs:
```logs
2020-10-21T13:17:04.098368695Z  2020-10-21T13:17:04.097Z zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext []                                                                                                                                                                                
2020-10-21T13:17:24.577373455Z  2020-10-21T13:17:24.576Z zigbee-herdsman:controller:endpoint Command 0xbc33acfffef4d2e6/1 genOnOff.on({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null})                                                                                                                                                                                                                                                  
2020-10-21T13:17:24.578576392Z  2020-10-21T13:17:24.577Z zigbee-herdsman:adapter:zStack:adapter sendZclFrameToEndpointInternal 0xbc33acfffef4d2e6:46526/1 (0,0,1)                                                                                                                                   
2020-10-21T13:17:24.578674594Z  2020-10-21T13:17:24.578Z zigbee-herdsman:adapter:zStack:znp:SREQ --> AF - dataRequest - {"dstaddr":46526,"destendpoint":1,"srcendpoint":1,"clusterid":6,"transid":83,"options":0,"radius":30,"len":3,"data":{"type":"Buffer","data":[1,58,1]}}                      
2020-10-21T13:17:24.579784810Z  2020-10-21T13:17:24.578Z zigbee-herdsman:adapter:zStack:unpi:writer --> frame [254,13,36,1,190,181,1,1,6,0,83,0,30,3,1,58,1,81]                                                                                                                                     
2020-10-21T13:17:24.582077409Z  2020-10-21T13:17:24.581Z zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,1,100,1,0,100]                                                                                                                                                                         
2020-10-21T13:17:24.582193981Z  2020-10-21T13:17:24.581Z zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,1,100,1,0,100]                                                                                                                                                               
2020-10-21T13:17:24.582431420Z  2020-10-21T13:17:24.581Z zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 1 - 3 - 4 - 1 - [0] - 100                                                                                                                                                            
2020-10-21T13:17:24.582505400Z  2020-10-21T13:17:24.582Z zigbee-herdsman:adapter:zStack:znp:SRSP <-- AF - dataRequest - {"status":0}                                                                                                                                                                
2020-10-21T13:17:24.582615416Z  2020-10-21T13:17:24.582Z zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext []                                                                                                                                                                                
2020-10-21T13:17:34.583898825Z  2020-10-21T13:17:34.583Z zigbee-herdsman:adapter:zStack:adapter Data confirm error (0xbc33acfffef4d2e6:46526,9999,0)                                                                                                                                                
2020-10-21T13:17:34.598129853Z  2020-10-21T13:17:34.583Z zigbee-herdsman:controller:endpoint Command 0xbc33acfffef4d2e6/1 genOnOff.on({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) failed (Data request failed with error: 'Timeout' (9999))                                                                                                                                                                                        
2020-10-21T13:18:01.293898723Z  2020-10-21T13:18:01.293Z zigbee-herdsman:controller:endpoint Command 0x60a423fffe9d9771/1 genOnOff.off({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null})                                                                                                                                                                                                                                                 
2020-10-21T13:18:01.294864461Z  2020-10-21T13:18:01.294Z zigbee-herdsman:adapter:zStack:adapter sendZclFrameToEndpointInternal 0x60a423fffe9d9771:64329/1 (0,0,1)                                                                                                                                   
2020-10-21T13:18:01.296356985Z  2020-10-21T13:18:01.295Z zigbee-herdsman:adapter:zStack:znp:SREQ --> AF - dataRequest - {"dstaddr":64329,"destendpoint":1,"srcendpoint":1,"clusterid":6,"transid":84,"options":0,"radius":30,"len":3,"data":{"type":"Buffer","data":[1,59,0]}}                      
2020-10-21T13:18:01.297105247Z  2020-10-21T13:18:01.296Z zigbee-herdsman:adapter:zStack:unpi:writer --> frame [254,13,36,1,73,251,1,1,6,0,84,0,30,3,1,59,0,239]                                                                                                                                     
2020-10-21T13:18:01.300702727Z  2020-10-21T13:18:01.300Z zigbee-herdsman:adapter:zStack:unpi:parser <-- [254,1,100,1,0,100]                                                                                                                                                                         
2020-10-21T13:18:01.301120144Z  2020-10-21T13:18:01.300Z zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext [254,1,100,1,0,100]                                                                                                                                                               
2020-10-21T13:18:01.302317452Z  2020-10-21T13:18:01.301Z zigbee-herdsman:adapter:zStack:unpi:parser --> parsed 1 - 3 - 4 - 1 - [0] - 100                                                                                                                                                            
2020-10-21T13:18:01.305446237Z  2020-10-21T13:18:01.305Z zigbee-herdsman:adapter:zStack:znp:SRSP <-- AF - dataRequest - {"status":0}                                                                                                                                                                
2020-10-21T13:18:01.306679656Z  2020-10-21T13:18:01.306Z zigbee-herdsman:adapter:zStack:unpi:parser --- parseNext []                                                                                                                                                                                
2020-10-21T13:18:11.322407332Z  2020-10-21T13:18:11.321Z zigbee-herdsman:adapter:zStack:adapter Data confirm error (0x60a423fffe9d9771:64329,9999,0)                                                                                                                                                
2020-10-21T13:18:11.322520330Z  2020-10-21T13:18:11.321Z zigbee-herdsman:controller:endpoint Command 0x60a423fffe9d9771/1 genOnOff.off({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) failed (Data request failed with error: 'Timeout' (9999))
```

zigbee2mqtt logs:
```log
INFO: MQTT publish: topic 'zigbee2mqtt/Bedroom Main Light Switch', payload '{"device":{"applicationVersion":65,"dateCode":"","friendlyName":"Bedroom Main Light Switch","hardwareVersion":1,"ieeeAddr":"0x60a423fffe9d9771","manufacturerID":4098,"manufacturerName":"_TYZB01_xfpdrwvc","model":"TS0011","networkAddress":64329,"powerSource":"Battery","stackVersion":0,"type":"EndDevice","zclVersion":3},"elapsed":355,"last_seen":1603286017742,"linkquality":84,"state":"OFF"}'
ERROR: No converter available for 'transition' (0)
ERROR: Publish 'set' 'state' to 'Bathroom Main Light Switch' failed: 'Error: Command 0xbc33acfffef4d2e6/1 genOnOff.on({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) failed (Data request failed with error: 'Timeout' (9999))'
INFO: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'set' 'state' to 'Bathroom Main Light Switch' failed: 'Error: Command 0xbc33acfffef4d2e6/1 genOnOff.on({}, {\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":false,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null}) failed (Data request failed with error: 'Timeout' (9999))'","meta":{"friendly_name":"Bathroom Main Light Switch"},"type":"zigbee_publish_error"}'
ERROR: No converter available for 'transition' (0)
ERROR: Publish 'set' 'state' to 'Bathroom Main Light Switch' failed: 'Error: Command 0xbc33acfffef4d2e6/1 genOnOff.off({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) failed (Data request failed with error: 'Timeout' (9999))'
INFO: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'set' 'state' to 'Bathroom Main Light Switch' failed: 'Error: Command 0xbc33acfffef4d2e6/1 genOnOff.off({}, {\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":false,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null}) failed (Data request failed with error: 'Timeout' (9999))'","meta":{"friendly_name":"Bathroom Main Light Switch"},"type":"zigbee_publish_error"}'
INFO: MQTT publish: topic 'zigbee2mqtt/Aqara Hall Motion Sensor 1', payload '{"battery":100,"device":{"applicationVersion":5,"dateCode":"20170627","friendlyName":"Aqara Hall Motion Sensor 1","hardwareVersion":1,"ieeeAddr":"0x00158d00045fc9f7","manufacturerID":4151,"manufacturerName":"LUMI","model":"RTCGQ11LM","networkAddress":22981,"powerSource":"Battery","softwareBuildID":"3000-0001","stackVersion":2,"type":"EndDevice","zclVersion":1},"elapsed":23001,"illuminance":0,"illuminance_lux":0,"last_seen":1603286011139,"linkquality":115,"occupancy":false,"voltage":3025}'
INFO: MQTT publish: topic 'zigbee2mqtt/Wardrobe LED1', payload '{"brightness":3,"color":{"hue":190,"saturation":31,"x":0.315764701278241,"y":0.325793955783666},"color_temp":157,"device":{"applicationVersion":32,"dateCode":"20200312","friendlyName":"Wardrobe LED1","hardwareVersion":1,"ieeeAddr":"0x680ae2fffefad577","manufacturerID":4476,"manufacturerName":"IKEA of Sweden","model":"LED1537R6/LED1739R5","networkAddress":13335,"powerSource":"Mains (single phase)","softwareBuildID":"2.0.029","stackVersion":98,"type":"Router","zclVersion":2},"last_seen":1603217187891,"linkquality":115,"state":"OFF","update":{"state":"idle"},"update_available":false}'
INFO: MQTT publish: topic 'zigbee2mqtt/Wardrobe LEDs', payload '{"brightness":3,"color":{"hue":190,"saturation":31,"x":0.315764701278241,"y":0.325793955783666},"color_temp":157,"state":"OFF"}'
INFO: MQTT publish: topic 'zigbee2mqtt/Wardrobe LED2', payload '{"brightness":3,"color":{"hue":190,"saturation":31,"x":0.315764701278241,"y":0.325793955783666},"color_temp":157,"device":{"applicationVersion":32,"dateCode":"20200312","friendlyName":"Wardrobe LED2","hardwareVersion":1,"ieeeAddr":"0x588e81fffe0fbe2a","manufacturerID":4476,"manufacturerName":"IKEA of Sweden","model":"LED1537R6/LED1739R5","networkAddress":54332,"powerSource":"Mains (single phase)","softwareBuildID":"2.0.029","stackVersion":98,"type":"Router","zclVersion":2},"last_seen":1603217072886,"linkquality":99,"state":"OFF","update":{"state":"idle"},"update_available":false}'
ERROR: Publish 'set' 'state' to 'Bedroom Main Light Switch' failed: 'Error: Command 0x60a423fffe9d9771/1 genOnOff.on({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) failed (Data request failed with error: 'Timeout' (9999))'
INFO: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'set' 'state' to 'Bedroom Main Light Switch' failed: 'Error: Command 0x60a423fffe9d9771/1 genOnOff.on({}, {\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":false,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null}) failed (Data request failed with error: 'Timeout' (9999))'","meta":{"friendly_name":"Bedroom Main Light Switch"},"type":"zigbee_publish_error"}'
ERROR: No converter available for 'transition' (0)
INFO: MQTT publish: topic 'zigbee2mqtt/Kitchen Main Light Switch', payload '{"device":{"applicationVersion":65,"dateCode":"","friendlyName":"Kitchen Main Light Switch","hardwareVersion":1,"ieeeAddr":"0xbc33acfffe770f76","manufacturerID":4098,"manufacturerName":"_TZ3000_9hpxg80k","model":"TS0011","networkAddress":56047,"powerSource":"Battery","stackVersion":0,"type":"EndDevice","zclVersion":3},"last_seen":1603286139803,"linkquality":36,"state":"ON"}'
ERROR: No converter available for 'transition' (0)
INFO: MQTT publish: topic 'zigbee2mqtt/Kitchen Main Light Switch', payload '{"device":{"applicationVersion":65,"dateCode":"","friendlyName":"Kitchen Main Light Switch","hardwareVersion":1,"ieeeAddr":"0xbc33acfffe770f76","manufacturerID":4098,"manufacturerName":"_TZ3000_9hpxg80k","model":"TS0011","networkAddress":56047,"powerSource":"Battery","stackVersion":0,"type":"EndDevice","zclVersion":3},"elapsed":130100,"last_seen":1603286139814,"linkquality":39,"state":"ON"}'
ERROR: No converter available for 'transition' (0)
INFO: MQTT publish: topic 'zigbee2mqtt/Kitchen Main Light Switch', payload '{"device":{"applicationVersion":65,"dateCode":"","friendlyName":"Kitchen Main Light Switch","hardwareVersion":1,"ieeeAddr":"0xbc33acfffe770f76","manufacturerID":4098,"manufacturerName":"_TZ3000_9hpxg80k","model":"TS0011","networkAddress":56047,"powerSource":"Battery","stackVersion":0,"type":"EndDevice","zclVersion":3},"last_seen":1603286142361,"linkquality":39,"state":"OFF"}'
INFO: MQTT publish: topic 'zigbee2mqtt/Kitchen Main Light Switch', payload '{"device":{"applicationVersion":65,"dateCode":"","friendlyName":"Kitchen Main Light Switch","hardwareVersion":1,"ieeeAddr":"0xbc33acfffe770f76","manufacturerID":4098,"manufacturerName":"_TZ3000_9hpxg80k","model":"TS0011","networkAddress":56047,"powerSource":"Battery","stackVersion":0,"type":"EndDevice","zclVersion":3},"elapsed":2560,"last_seen":1603286142368,"linkquality":36,"state":"OFF"}'
ERROR: Publish 'set' 'state' to 'Bedroom Main Light Switch' failed: 'Error: Command 0x60a423fffe9d9771/1 genOnOff.off({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) failed (Data request failed with error: 'Timeout' (9999))'
INFO: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'set' 'state' to 'Bedroom Main Light Switch' failed: 'Error: Command 0x60a423fffe9d9771/1 genOnOff.off({}, {\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":false,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null}) failed (Data request failed with error: 'Timeout' (9999))'","meta":{"friendly_name":"Bedroom Main Light Switch"},"type":"zigbee_publish_error"}'
ERROR: Publish 'set' 'state' to 'Bedroom Main Light Switch' failed: 'Error: Command 0x60a423fffe9d9771/1 genOnOff.on({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) failed (Data request failed with error: 'Timeout' (9999))'
INFO: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'set' 'state' to 'Bedroom Main Light Switch' failed: 'Error: Command 0x60a423fffe9d9771/1 genOnOff.on({}, {\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":false,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null}) failed (Data request failed with error: 'Timeout' (9999))'","meta":{"friendly_name":"Bedroom Main Light Switch"},"type":"zigbee_publish_error"}'
ERROR: No converter available for 'transition' (0)
ERROR: Publish 'set' 'state' to 'Bathroom Main Light Switch' failed: 'Error: Command 0xbc33acfffef4d2e6/1 genOnOff.on({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) failed (Data request failed with error: 'Timeout' (9999))'
INFO: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'set' 'state' to 'Bathroom Main Light Switch' failed: 'Error: Command 0xbc33acfffef4d2e6/1 genOnOff.on({}, {\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":false,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null}) failed (Data request failed with error: 'Timeout' (9999))'","meta":{"friendly_name":"Bathroom Main Light Switch"},"type":"zigbee_publish_error"}'
ERROR: No converter available for 'transition' (0)
ERROR: No converter available for 'transition' (0)
ERROR: Publish 'set' 'state' to 'Bedroom Main Light Switch' failed: 'Error: Command 0x60a423fffe9d9771/1 genOnOff.off({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null}) failed (Data request failed with error: 'Timeout' (9999))'
INFO: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"Publish 'set' 'state' to 'Bedroom Main Light Switch' failed: 'Error: Command 0x60a423fffe9d9771/1 genOnOff.off({}, {\"timeout\":10000,\"disableResponse\":false,\"disableRecovery\":false,\"disableDefaultResponse\":false,\"direction\":0,\"srcEndpoint\":null,\"reservedBits\":0,\"manufacturerCode\":null,\"transactionSequenceNumber\":null}) failed (Data request failed with error: 'Timeout' (9999))'","meta":{"friendly_name":"Bedroom Main Light Switch"},"type":"zigbee_publish_error"}'
INFO: MQTT publish: topic 'zigbee2mqtt/Aqara Wardrobe Door Sensor', payload '{"battery":100,"contact":false,"device":{"applicationVersion":3,"dateCode":"20161128","friendlyName":"Aqara Wardrobe Door Sensor","hardwareVersion":2,"ieeeAddr":"0x00158d00042d1906","manufacturerID":4151,"manufacturerName":"LUMI","model":"MCCGQ11LM","networkAddress":8937,"powerSource":"Battery","softwareBuildID":"3000-0001","stackVersion":2,"type":"EndDevice","zclVersion":1},"last_seen":1603286586253,"linkquality":94,"voltage":3005}'
```

![image](https://user-images.githubusercontent.com/2881159/96725858-aac35800-13b1-11eb-9a1c-82adf0e97d20.png)

**Comment by Koenkk** [comment]
@Shaquu when the switches stopped working, do they show no connection on the networkmap? (is it kitchen/bedroom main light switch?)

**Comment by Shaquu** [comment]
@Koenkk yes, they are on the map. It is main switch for bedroom and bathroom I believe

**Comment by Konsts** [comment]
> @Shaquu, когда переключатели перестали работать, они не показывают соединение на карте сети? (это главный выключатель света в кухне / спальне?)

Confirm? i have same problem with TuYa TS0012 2gang swithes.
The state changes when the switch is pressed
but not controlled remotely
Always occurs after reboot z2m

**Comment by Shaquu** [comment]
@Konsts indeed. In my case they are 1 gang.
They work for day or couple days and then stops responding to remote control (via Home.app => NRCHKB => node-red => zigbee2mqtt)
But yes, they are visible everywhere.

**Comment by Koenkk** [comment]
> @Koenkk yes, they are on the map. It is main switch for bedroom and bathroom I believe

But do they still show as connected on the map after they **stopped** working?

**Comment by Shaquu** [comment]
> > @Koenkk yes, they are on the map. It is main switch for bedroom and bathroom I believe
> 
> 
> 
> But do they still show as connected on the map after they **stopped** working?

Yes, it is on the map. Bedroom switch LQI 84

**Comment by Koenkk** [comment]
@Shaquu would you able to sniff the traffic for the following two scenarios (https://www.zigbee2mqtt.io/how_tos/how_to_sniff_zigbee_traffic.html):
- Controlling works (send e.g. on command): 
- Stopped working

**Comment by Shaquu** [comment]
@Koenkk question to you. Do I have to sniff with the same stick my zigbee devices are connected with?
Or can I sniff other stick? If second than there is no logs. If first then I am unable to start zigbee2mqtt on my Win10 to pair devices.

**Comment by Shaquu** [comment]
Possibly related with #4150

**Comment by Koenkk** [comment]
You need to sniff with an other stick.

**Comment by Shaquu** [comment]
@Koenkk no results while sniffing:
![image](https://user-images.githubusercontent.com/2881159/97460622-36eaf780-193d-11eb-87a9-f3f1a583e447.png)

**Comment by Koenkk** [comment]
Make sure you sniff on the correct channel, you should se traffic even when you don't execute a command.

**Comment by domnich** [comment]
> Thanks Koenkk. After my previous comment I added property "disableDefaultResponse: ture" into Blitzwolf shp-13 object to the "meta" (in devices.js) and for now everything works Ok...I can even say that my wall sockets now connected to this router. At the moment I can't say for sure that my fix fixed the problem, as little time has passed. Now I want to watch and if the error will appears again I will necessarily sniff my traffic and provide it to you.
> 
> Koenkk could you please share some good resources with detailed info about zigbee device object model (e.g. disableDefaultResponse etc) ? I will be very grateful.

Just a minor update from my side... Since I added property "disableDefaultResponse: ture" into Blitzwolf shp-13 everything works OK in my case, was not able to reproduce thia bug anymore..

**Comment by Shaquu** [comment]
> Make sure you sniff on the correct channel, you should se traffic even when you don't execute a command.

According to z2m UI it is channel 11.
![image](https://user-images.githubusercontent.com/2881159/97636728-d55c8300-1a39-11eb-9f5a-444b7c38943b.png)

**Comment by Koenkk** [comment]
@Shaquu then something is wrong with the sniffer setup, I see you use ZBOSS, maybe you can try with a CC2531.

**Comment by Shaquu** [comment]
@Koenkk I am using ZBOSS gui app according to https://www.zigbee2mqtt.io/how_tos/how_to_sniff_zigbee_traffic.html
My stick is CC2531 as you said.

**Comment by Koenkk** [comment]
I see, not sure what is wrong with your setup. You might want to try with linux/wirehsark.

**Comment by biejay** [comment]
Hey @domnich 
it's a while ago but how did you edit your devices.js to `disableDefaultResponse: true` ?

I run Zigbee2mqtt Addon on Hassio and my multiple TS0121_plug and TS011F_plug_3 sometimes don't toggle or only after a really long time. So I guess I got the same problem as you.

Thanks in advance!

**Comment by rdl987** [comment]
Hello,
I have the same problem but in my case it's regarding all devices, it became unresponsive.

I'm using zigbee2mqtt on docker running in a qnap nas and a SONOFF Zigbee 3.0 USB Dongle Plus ZBDongle-P.

Could anyone help me?
Thanks

**Comment by KostyaSha** [comment]
Seems starts failing with the latest update

**Comment by marcusletric** [comment]
I have this exact problem but my setup is way different. 
I got a SonOff ZB BRIDGE-P that I put tasmota on, also got a USB zigbee repeater

My HA as well as Z2M and Mosquitto on docker. I get this issue with a Moes light switch TS0011 and Moes TRV too TS0601, when I operate the devices the status updates on HA but when I remotely try to change state from HA, I get this error.

Additionally when I rearranged my network and gave some space for zigbee channels and moved coordinator and repeater around, this issue appeared less often, but still does.

Unpairing and re-pairing solves the issue for a while.

I haven't tried disable default response but I don't know how that can be done, should that solve my issue for good? Why is this even happening in the first place?

**Comment by rdl987** [comment]
Good morning, 
I tried to downgrade to the 20221226 firmware but this solve only for a while.

Could adding also `disableDefaultResponse: true` solve?

**Comment by Macht59** [comment]
I have the same issue with SONOFF S31ZB

```
Error 2024-02-20 20:37:01Publish 'set' 'state' to 'Knit Lamp' failed: 'Error: Command 0x00124b002a634909/1 genOnOff.off({}, {"timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Data request failed with error: 'MAC no ack' (233))'
```

**Comment by raadms** [comment]
> Thanks Koenkk. After my previous comment I added property "disableDefaultResponse: ture" into Blitzwolf shp-13 object to the "meta" (in devices.js) and for now everything works Ok...I can even say that my wall sockets now connected to this router. At the moment I can't say for sure that my fix fixed the problem, as little time has passed. Now I want to watch and if the error will appears again I will necessarily sniff my traffic and provide it to you.
> 
> Koenkk could you please share some good resources with detailed info about zigbee device object model (e.g. disableDefaultResponse etc) ? I will be very grateful.

Thank @domnich, 

I have the same issue can you please explain where can i find that file and what shall I add. much appreciate it. 


thnx
